### PR TITLE
Mark phishing zero txs as ignored

### LINKED ISSUE #5749: Handle the 0 token transfer phishing scams
## Problem Definition

There is this phishing scam where someone can make it appear as if you transferred tokens (USDC and more) out of your wallet to a specific address.

More info: https://cointelegraph.com/news/scammers-are-targeting-crypto-users-with-new-zero-value-transferfrom-trick

Unfortunately our UI is also a victim to that. Case in point for our very own rotki.eth

The transaction here: https://etherscan.io/tx/0xb45ef1a202a8d9e983cf59129d28f79057969bb822f62e4b7d9f1ac8853d23ed

How this works is quite silly and it's a flaw of many token contracts, including USDC. The `transferFrom` function can be called by anyone as long as the approval of the address is enough for the amount. So the attacker's address has approval for 0 amount (the default), so they can initiate a transfer of 0 amount of tokens. And this transfer is valid and logged on-chain.

## Task

I think for us the fix is easy. Zero value token transfers are useless. Even if you actually made them yourself. I would opt for just not generating an event in that case. And somehow also ignoring the transaction (as the transaction would be a transaction with no events).

### PR DESCRIPTION
**Pull request by nebolax**
Closes #5749

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

**Review by prettyirrelevant — COMMENTED** [review]
Just a comment.

File: rotkehlchen/chain/evm/decoding/decoder.py [hunk 1]
```diff
@@ -325,6 +329,13 @@ def decode_transaction(
                 write_cursor=write_cursor,
                 history=events,
             )
+            if all([event.balance.amount == ZERO for event in events]) is True:
```
[the remark this answers is not in the document]
Comment (nebolax) [reply rotkehlchen/chain/evm/decoding/decoder.py:332]: Yeah, looks true, thanks.

File: rotkehlchen/chain/evm/decoding/decoder.py [hunk 2]
```diff
@@ -325,6 +330,13 @@ def decode_transaction(
                 write_cursor=write_cursor,
                 history=events,
             )
+            if all([event.balance.amount == ZERO and event.asset != A_ETH for event in events]) is True:  # noqa: E501
```
Comment (LefterisJP) [inline rotkehlchen/chain/evm/decoding/decoder.py:333]: I would not do it like this. This is a hack and does not fit the decoding process. Specifically becuase it adds another implicit post-decoding rule, which takes linearly more time, the more events a transaction has.

What you need to do is:
1. If a **token** transfer is 0 value, not decode it at all, in the token transfers decoding.
2. Then here check that the number of events of is empty. If yes ignore the transaction, saying it's most probably a phishing transaction with the link you already have as comment.

**Review by LefterisJP — APPROVED** [review]
LGTM!

**Pull request merged**